example/spec/parallel/dbRecords/DeadLetterArchiveProcessingSpec.js

```diff
@@ -91,10 +92,10 @@ describe('A dead letter record archive processing operation', () => {
         fakeFileFactory({ bucket: systemBucket }),
         fakeFileFactory({ bucket: systemBucket }),
       ];
-
+      collectionId = constructCollectionId(testCollection.name, testCollection.version);
```

Comment:
I think we should URI encode the version everywhere whether we do it in a function like `constructCollectionId` or not